applications/serial_lte_modem/doc/GNSS_AT_commands.rst

```diff
@@ -352,7 +175,7 @@ The ``#XAGPS`` command runs the GNSS together with the nRF Cloud A-GPS service.
 .. note::
    To use ``#XAGPS``, the following preconditions apply:
 
-   * You must define :ref:`CONFIG_SLM_AGPS <CONFIG_SLM_AGPS>`.
+   * You must define :ref:`CONFIG_SLM_NRF_CLOUD <CONFIG_SLM_NRF_CLOUD>` and :ref:`CONFIG_NRF_CLOUD_AGPS <CONFIG_NRF_CLOUD_AGPS>`.
    * You must have access to nRF Cloud through the LTE network for receiving A-GPS data.
```

Comment:
@wiba-nordic CONFIG_NRF_CLOUD_AGPS is not defined in SLM. How to address below doc build error?

Warning, treated as error:
/home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrf/src/applications/serial_lte_modem/doc/GNSS_AT_commands.rst:178:undefined label: 'config_nrf_cloud_agps'
FAILED: CMakeFiles/nrf-all /home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/CMakeFiles/nrf-all

Reply:
It can be linked to like most Kconfig options using :kconfig:option:`CONFIG_NRF_CLOUD_AGPS`

Reply:
Thanks ☕